rest/authorization-server/src/main/resources/application.yml

```diff
@@ -10,20 +10,34 @@
 server:
   port: 8090
 
+couchdb:
+  url: http://sw360couchdb:5984
```

Comment:
these default settings only work with docker, and not vagrant?

Reply:
This is possible, I never used the vagrant setup. But is there then a way to support both?